harness/determined/cli/workspace.py

```diff
@@ -145,6 +147,37 @@ def list_pools(args: argparse.Namespace) -> None:
         as_csv=False,
     )
 
+def add_workspace_namespace_binding(args: argparse.Namespace):
+    sess = cli.setup_session(args)
+    w = api.workspace_by_name(sess, args.workspace_name)
+    content= bindings.v1ModifyWorkspaceNamespaceBindingRequest(
+        id=w.id,
+        autoCreateNamespace=args.auto_create_namespace,
+        clusterName=args.cluster_name,
+        namespaceName=args.namespace_name,
+    )
+    workspace_name = str(args.workspace_name)
+    
+    n = bindings.post_ModifyWorkspaceNamespaceBinding(
+        sess, 
+        body=content,
+        id=w.id).name
+
+    print("Workspace " + workspace_name + " is bound to " + n)
+    return None
+
+def set_quota(args: argparse.Namespace):
```

Comment:
could you add an e2e to e2e_tests/tests/cluster/test_workspace_org.py for a test workflow that includes these functions? 
do we need to wait until backend functionality is implemented or could you faux-implement it for the test?

Reply:
Backend functionality is implemented in this PR! 
Following conversations from this week, I'll add the remaining tests (including e2e tests) in a separate PR.